vector/src/main/res/layout/fragment_timeline.xml

```diff
@@ -37,13 +37,6 @@
 
     </com.google.android.material.appbar.AppBarLayout>
 
-    <androidx.constraintlayout.widget.Barrier
-        android:id="@+id/timelineRecyclerViewBarrier"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:barrierDirection="top"
-        app:constraint_referenced_ids="composerLayout,notificationAreaView,failedMessagesWarningStub" />
```

Comment:
this was duplicated code with `badgeBarrier` which has been renamed to `bottomBarrier`